proj/src/store.js

```diff
@@ -1,19 +1,18 @@
 import { createStore, compose, applyMiddleware} from 'redux'
-import thunk from 'redux-thunk';
-import persistState from 'redux-localstorage'
+// import persistState from 'redux-localstorage'
 
 import rootReducer from 'reducers/root'
-
-const middleWare = [thunk]
+import projectsMiddleware from 'middleware/projectsMiddleware';
```

Comment:
projectsMiddleware? isn't it apiMiddleware? please go over all the changes before submitting a PR. seems like you were in a hurry :(